addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -192,3 +193,75 @@ test("mention suggestions in thread match channel restrictions", async () => {
     await contains(".o-mail-Composer-suggestion", { text: "Mitchell Admin" });
     await contains(".o-mail-Composer-suggestion", { text: "p1" });
 });
+
+test("sub channel is muted (opacity-50) when mute is active", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-mail-Discuss-threadName", { value: "General" });
+    await click("button[title='Threads']");
```

Comment:
```js
await contains(".o-mail-Discuss-threadName", { value: "General" });
```
is not useful. This is already tested and there's no gain in awaiting this: the
```js
await contains("button[title='Thread']")
```

Is already waiting the necessary time for click.